core/src/lib/util.cc

```diff
@@ -85,6 +85,15 @@ void EscapeString(PoolMem& snew, const char* old, int len)
   *n = 0;
 }
 
+
+std::string EscapeString(const char* old)
+{
+  PoolMem snew;
+  EscapeString(snew, old, strlen(old));
+  return std::string(snew.c_str());
```

Comment:
This returns a copy of the local string and thats ok. 

However, I'd prefer to avoid PoolMem and use a std::vector<char> instead. In turn this needs a slight refactoring of the EscapeString(PoolMem,..) function but would eliminate another occurrence of PoolMem.